apps/guardian-ui/src/components/VerifyGuardians.tsx

```diff
@@ -334,14 +356,33 @@ export const VerifyGuardians: React.FC<Props> = ({ next }) => {
           <Button
             isDisabled={!verifiedConfigs}
             isLoading={isStarting}
-            onClick={verifiedConfigs ? handleNext : undefined}
+            onClick={handleNext}
             leftIcon={<Icon as={ArrowRightIcon} />}
             width={{ base: 'full', sm: 'auto' }}
           >
             {t('common.next')}
           </Button>
           <WaitingForVerification verifiedConfigs={verifiedConfigs} />
         </Stack>
+        <Modal isOpen={isOpen} onClose={() => setIsOpen(false)}>
+          <ModalOverlay />
+          <ModalContent>
+            <ModalCloseButton />
+            <ModalBody>{t('verify-guardians.leader-confirm-done')}</ModalBody>
```

Comment:
wrong path to locale:
```suggestion
            <ModalBody>{t('setup.progress.verify-guardians.leader-confirm-done')}</ModalBody>
```